eventsources/sources/resource/start.go

```diff
@@ -250,7 +250,14 @@ func LabelReq(sel v1alpha1.Selector) (*labels.Requirement, error) {
 	if sel.Operation != "" {
 		op = selection.Operator(sel.Operation)
 	}
-	req, err := labels.NewRequirement(sel.Key, op, []string{sel.Value})
+	values := []string{sel.Value}
```

Comment:
Start with `var values = make([]string, 0)`, and use `if` `else`?

Reply:
Makes sense will make the change

Reply:
Used ```switch``` and ```var []string``` instead of if else block due to linting error.